### PR TITLE
Add "Spatie Media Library Pro" detection

### PR DESCRIPTION
**Pull request by osbre**
Websites to test:
- https://medialibrary.pro/demo-attachment

### CODE REVIEW

File: src/technologies/s.json [hunk 1]
```diff
@@ -4793,6 +4793,22 @@
...
+    "description": "A set of customizable UI components for Spatie Media Library.",
+    "icon": "Spatie.png",
+    "implies": [
+      "Laravel"
+    ],
+    "pricing": [
+      "onetime",
+      "low"
+    ],
+    "scripts": "media\\-library\\-pro\\-core",
```
Comment (nurbek91) [inline src/technologies/s.json:4809]: I don't see such a file.
image

File: src/technologies/s.json [hunk 2]
```diff
@@ -4793,6 +4793,22 @@
     "icon": "SparkPost.svg",
     "website": "https://www.sparkpost.com/"
   },
+  "Spatie Media Library Pro": {
+    "cats": [
+      66
+    ],
+    "description": "A set of customizable UI components for Spatie Media Library.",
```
Comment (nurbek91) [inline src/technologies/s.json:4800]: Spatie Media Library Pro is a ....

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, are there any other examples besides the demo site?

File: src/technologies/s.json (hunk 1 above)
Comment (osbre) [reply src/technologies/s.json:4809]: It's not a file (for files `scriptSrc` is used), but a specific word in the file that is going to be included with any JS build that has this media-library-pro component included.
image

**Review by nurbek91 — APPROVED** [review]

**Pull request merged**